surrogate/thr_sq.py

```diff
@@ -14,12 +14,20 @@ def _thr_sq_encode(
     sq_factor,          # quantization factor
     rectify_negatives,  # whether to apply crelu
     l2_normalize,       # whether to l2-normalize vectors
+    rotation_matrix,    # rotation matrix used to rotate features
+    mean,
```

Comment:
could you add a brief comment also of the 'mean' parameter?

Reply:
yes of course, added it in b01cc798137e381002c7294e956938cb48609a7d